app/src/main/java/com/example/vshcheglov/webshop/presentation/helpers/Router.kt

```diff
@@ -0,0 +1,40 @@
+package com.example.vshcheglov.webshop.presentation.helpers
+
+import android.content.Context
+import android.content.Intent
+import com.example.vshcheglov.webshop.presentation.basket.BasketActivity
+import com.example.vshcheglov.webshop.presentation.login.LoginActivity
+import com.example.vshcheglov.webshop.presentation.main.MainActivity
+import com.example.vshcheglov.webshop.presentation.order.OrderActivity
+import com.example.vshcheglov.webshop.presentation.purchase.PurchaseActivity
+import com.example.vshcheglov.webshop.presentation.registration.RegisterActivity
+
+object Router {
+    fun navigateToMainActivity(context: Context) {
```

Comment:
as an example
`fun showMain(activity: Activity?) {
        activity?.apply {
            val intent = Intent(this, MainActivity::class.java).apply {
                flags = Intent.FLAG_ACTIVITY_CLEAR_TASK or
                        Intent.FLAG_ACTIVITY_NEW_TASK
            }
            startActivity(intent)
        }
    }`